### PR TITLE
Check summary [v2]

### PR DESCRIPTION
**Pull request by richtja**
Output of selftests check with summary of all failed tests.

Signed-off-by: Jan Richter <jarichte@redhat.com>

Now the check output will look like this:
```
...
 (check-852/956) selftests/lint.sh: FAIL (91.79 s)
RESULTS    : PASS 956 | ERROR 1 | FAIL 3 | SKIP 44 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/jarichte/avocado/job-results/job-2021-03-10T15.40-f7a9604/results.html
JOB TIME   : 462.65 s
Failed tests:
check-417-selftests/unit/test_test.py:MockingTest.test_combination_2  ERROR
check-416-selftests/unit/test_test.py:MockingTest.test_combination  FAIL
check-772-selftests/functional/test_loader.py:LoaderTestFunctional.test_python_unittest  FAIL
check-852-selftests/lint.sh  FAIL
```
---
Changes form v1 #4448:

Usage of Job API instead of journal plugin

### CODE REVIEW

File: selftests/check.py [hunk 1]
```diff
@@ -518,6 +518,13 @@ def create_suites(args):
     return suites
 
 
+def print_failed_tests(tests):
+    if tests:
```
Comment (beraldoleal) [inline selftests/check.py:522]: `get_failed_tests()` is returning always a list. So looks like, this `if` is unnecessary.

**Review by beraldoleal — COMMENTED** [review]
Hi @richtja , besides the small comment, this LGTM.

File: selftests/check.py (hunk 1 above)
Comment (richtja) [reply selftests/check.py:522]: Yes, sure

Comment (beraldoleal) [reply selftests/check.py:522]: Oh makes sense, I missed that.

**Review by beraldoleal — APPROVED** [review]
LGTM

**Review by willianrampazzo — APPROVED** [review]

**Pull request merged**